protected/humhub/modules/ui/form/widgets/ContentVisibilitySelect.php

```diff
@@ -41,7 +42,7 @@
 class ContentVisibilitySelect extends InputWidget
 {
     /**
-     * @var bool Automatically hide the field when no mulitple visibility modes available
+     * @var bool Automatically hide the field when no only one visibility option is available
```

Comment:
Probably could change this to something simpler
`Hide the field when only one visibility option is available` or
`Automatically hide the field when only one visibility option is available`

Reply:
Ok, good point!